### ISSUE TITLE
[1.19.2] Accumulator Backpack Upgrades Not Functioning

### ISSUE THREAD

**Issue by BowmasterE** [BluSunrize/ImmersiveEngineering#5841]
### Minecraft Version

1.19.2

### Immersive Engineering Version

1.19.2-9.2.3-167

### Expected Behavior

Accumulator backpack charges internal accumulator, put into it via the workbench, from wire network, using the Charging Antenna.

Accumulator backpack charges items not being held in main or off-hand while it has charge using the Induction Charger.

### Actual Behavior

Accumulator backpack does not charge when standing on the ground underneath uninsulated wire whilst wearing it.

Accumulator backpack does not charge items in inventory not being held in either hand.

### Steps to reproduce

1. Put expected upgrades into the accumulator backpack via the workbench
2.a Stand under uninsulated wire connected to a power source
2.b Have chargeable item in inventory with room to charge

### Debug Crash Log

https://gist.github.com/BowmasterE/488e3ddbb6468141621cc0f15e82d947

**Comment by BluSunrize** [comment]
Can you reproduce this in an instance with just IE installed?

**Comment by BowmasterE** [comment]
> Can you reproduce this in an instance with just IE installed?

Yes. I just installed a new instance with just IE installed, and ran into the same issue.

**Comment by BluSunrize** [comment]
Hmm, can you provide some pictures of the backpack, its tooltip, and the items you were trying to charge? (In the IE-only instance)

**Comment by BowmasterE** [comment]
![2024-01-05_15 32 31](https://github.com/BluSunrize/ImmersiveEngineering/assets/24629099/1ed90b43-ec13-42ff-876f-2c8cb118d584)
![2024-01-05_15 32 45](https://github.com/BluSunrize/ImmersiveEngineering/assets/24629099/6593f9a5-965c-41fd-b5b8-ab59b148c846)
![2024-01-05_15 32 52](https://github.com/BluSunrize/ImmersiveEngineering/assets/24629099/f087fe71-87c1-4279-9099-0b080f8a385b)
![2024-01-05_15 33 25](https://github.com/BluSunrize/ImmersiveEngineering/assets/24629099/5911febe-946c-42d4-9516-f672272c3aea)
![2024-01-05_15 33 50](https://github.com/BluSunrize/ImmersiveEngineering/assets/24629099/fef7cce3-973f-4462-af4d-3df1bec1ca77)
![2024-01-05_15 34 03](https://github.com/BluSunrize/ImmersiveEngineering/assets/24629099/3e068c9c-42b1-4ae5-9d37-2a35df1d0c95)
![2024-01-05_15 34 06](https://github.com/BluSunrize/ImmersiveEngineering/assets/24629099/14ad1b9f-e7d3-4fc4-8fe0-a0be8a875ed2)

**Comment by BluSunrize** [comment]
I have reproduced the induction charger not working, this is an issue affects 1.20 as well - poor coding from my end, I'll resolve that.

The antenna is working fine for me, however:
![grafik](https://github.com/BluSunrize/ImmersiveEngineering/assets/4106382/47bb4885-ed72-4b94-a486-9763fc8ad672)
It has connected to a wire and the backpack is being charged up.
Note however, that it can only connect to un-insulated wires that actually have energy running through them.

**Comment by BowmasterE** [comment]
By having energy running through them, do you mean being connected to an energy source or do you mean connected to both an energy source and something else consuming energy? I have used uninsulated LV wire and HV wire.

**Comment by BowmasterE** [comment]
I have just tried uninsulated MV wire as well, both just connected to an energy source and to both an energy source and a consumer, and still cannot get the charging antenna to work.

**Comment by BluSunrize** [comment]
Hmm, very odd. Can I ask you to make a video of the situation? I'm running out of ideas on how to debug this further :/
